### ISSUE TITLE
Booted method and global scope declaration

### ISSUE THREAD

**Issue by webbati** [wintercms/winter#747]
### Winter CMS Build

1.1

### PHP Version

7.4

### Database engine

MySQL/MariaDB

### Issue description

Global scopes documentation seem to refer only to Laravel 9.x (Winter 1.2.x), for previous version that are based on Laravel 6.x classes and methods are different.

### Steps to replicate

Create a global scope following the Winter documentation https://wintercms.com/docs/database/model#global-scopes

### Workaround

In the Scope class change type hinting from Winter to Illuminate class like this

from  
```php
use Winter\Storm\Database\Builder;
use Winter\Storm\Database\Model;
use Illuminate\Database\Eloquent\Scope;

class AncientScope implements Scope
{
 
    public function apply(Builder $builder, Model $model)
    {
        $builder->where('age', '>', 200);
    }
}
```

to  
```php
use Illuminate\Database\Eloquent\Builder;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Scope;

class AncientScope implements Scope
{
    
    public function apply(Builder $builder, Model $model)
    {
      $builder->where('age', '>', 200);
    }
}
```

and substitute booted method with boot method in the models that implements the global scope

from
```php
protected static function booted()
    {
        static::addGlobalScope(new AncientScope);
    }
``

to

protected static function boot()
{
    parent::boot();
    static::addGlobalScope(new AncientScope);
}
```

**Comment by webbati** [comment]
Added PR to Winter Docs

See https://github.com/wintercms/docs/pull/101

**Comment by bennothommo** [comment]
We've merged in the changes to the docs for the 1.1 branch. We'll just need to get that online on the new docs, which should hopefully be soon.
